.evergreen/ocsp/README.md

```diff
@@ -2,10 +2,12 @@
 
 The test certificates here were generating using the server team's [`mkcert.py`](https://github.com/mongodb/mongo/blob/master/jstests/ssl/x509/mkcert.py) tool.
 
-In order to generate a fresh set of certificates, clone the [`mongo` repository](https://github.com/mongodb/mongo) and run the following command from the root of the `mongo` repository:
+In order to generate a fresh set of certificates, clone this branch on my fork of the [`mongo` repository](https://github.com/vincentkam/mongo/tree/mkcert-ecdsa) and run the following command from the root of the `mongo` repository:
```

Comment:
Is this a temporary change? For style reasons, I think we should avoid using personalized language (e.g. "my fork") in documents.

Reply:
Ah okay, I've updated the language to say "a fork" until my PR to the mongo repo gets approved